src/definition.ts

```diff
@@ -156,13 +167,33 @@ class API {
       );
     }
 
-    if (!API.isOpenAPI(parsed) && !API.isAsyncAPI(parsed)) {
+    if (
+      !API.isOpenAPI(parsed) &&
+      !API.isAsyncAPI(parsed) &&
+      !API.isOpenAPIOverlay(parsed)
+    ) {
       throw new UnsupportedFormat();
     }
 
     return [raw, parsed];
   }
 
+  serializedDefinition(outputPath?: string): string {
```

Comment:
Maybe, to be consistant with the way we name thing, we should rename this into an action verb `serializeDefinition`.